Misc/NEWS.d/next/Library/2018-04-27-22-18-38.bpo-33336.T8rxn0.rst

```diff
@@ -0,0 +1,2 @@
+``imaplib`` now allows ``MOVE`` command in ``IMAP4.uid()`` and potentially
+as a name of supported method of ``IMAP4`` object.
```

Comment:
Would you mind to add a reference to the RFC 6851? Example:

``imaplib`` now allows ``MOVE`` command in ``IMAP4.uid()`` (RFC 6851: IMAP MOVE Extension)
and potentially as a name of supported method of ``IMAP4`` object.